src/main.lua

```diff
@@ -2057,72 +2069,66 @@ addEventingHandler("releaseName", utils.hasMatchingTag("Action", ActionMap.Relea
 	local removedRecord = arns.removeRecord(name)
 	local removedPrimaryNamesAndOwners = primaryNames.removePrimaryNamesForBaseName(name) -- NOTE: this should be empty if there are no primary names allowed before release
-	local createdAuction = arns.createAuction(name, timestamp, initiator)
-	local createAuctionData = {
+	local returnedName = arns.createReturnedName(name, timestamp, initiator)
+	local returnedNameData = {
 		removedRecord = removedRecord,
 		removedPrimaryNamesAndOwners = removedPrimaryNamesAndOwners,
-		auction = createdAuction,
+		returnedName = returnedName,
 	}
 
-	addAuctionResultFields(msg.ioEvent, {
+	addReturnedNameResultFields(msg.ioEvent, {
 		name = name,
-		auction = createAuctionData.createdAuction,
-		removedRecord = createAuctionData.removedRecord,
-		removedPrimaryNamesAndOwners = createAuctionData.removedPrimaryNamesAndOwners,
+		returnedName = returnedNameData.returnedName,
+		removedRecord = returnedNameData.removedRecord,
+		removedPrimaryNamesAndOwners = returnedNameData.removedPrimaryNamesAndOwners,
 	})
 
-	-- note: no change to token supply here - only on auction bids
-	msg.ioEvent:addField("Auctions-Count", utils.lengthOfTable(NameRegistry.auctions))
+	-- note: no change to token supply here - only on buy record of returned name
+	msg.ioEvent:addField("Returned-Name-Count", utils.lengthOfTable(NameRegistry.returned))
 	msg.ioEvent:addField("Records-Count", utils.lengthOfTable(NameRegistry.records))
 
-	local auction = {
+	local releaseNameData = {
 		name = name,
-		startTimestamp = createAuctionData.auction.startTimestamp,
-		endTimestamp = createAuctionData.auction.endTimestamp,
-		initiator = createAuctionData.auction.initiator,
-		baseFee = createAuctionData.auction.baseFee,
-		demandFactor = createAuctionData.auction.demandFactor,
-		settings = createAuctionData.auction.settings,
+		startTimestamp = returnedName.startTimestamp,
+		endTimestamp = returnedName.startTimestamp + constants.returnedNamePeriod,
+		initiator = returnedName.initiator,
 	}
 
 	-- send to the initiator and the process that released the name
 	Send(msg, {
 		Target = initiator,
-		Action = "Auction-Notice",
+		Action = "Returned-Name-Notice",
 		Name = name,
-		Data = json.encode(auction),
+		Data = json.encode(releaseNameData),
 	})
 	Send(msg, {
 		Target = processId,
-		Action = "Auction-Notice",
+		Action = "Returned-Name-Notice",
 		Name = name,
-		Data = json.encode(auction),
+		Data = json.encode(releaseNameData),
 	})
 end)
 
--- AUCTIONS
-addEventingHandler("auctions", utils.hasMatchingTag("Action", ActionMap.Auctions), function(msg)
+addEventingHandler(ActionMap.ReturnedNames, utils.hasMatchingTag("Action", ActionMap.ReturnedNames), function(msg)
 	local page = utils.parsePaginationTags(msg)
-	local auctions = arns.getAuctions()
-	local auctionsWithoutFunctions = {}
+	local returnedNames = arns.getReturnedNames()
+	local returnedNamesWithEndTimestampAndPremiumMultiplier = {}
```

Comment:
Can we get a gentler variable name if we apply a type annotation to this local? e.g.
```
--- @type MyThing
local myThing = {}
```

Reply:
94bfe2f

Reply:
Nit that `ReturnedNameReturn` is a little funny for a name. Maybe `ReturnedNameData`?

Reply:
0831b66